src/main/java/hudson/plugins/gradle/GradleInstallation.java

```diff
@@ -14,23 +24,18 @@
 import hudson.tools.ToolInstaller;
 import hudson.tools.ToolProperty;
 import jenkins.security.MasterToSlaveCallable;
-import org.jenkinsci.Symbol;
-import org.kohsuke.stapler.DataBoundConstructor;
-
-import javax.inject.Inject;
-import java.io.File;
-import java.io.IOException;
-import java.io.Serializable;
-import java.util.Collections;
-import java.util.List;
 
 
 /**
+ * Gradle Installation in Jenkins.
+ * 
  * @author Gregory Boissinot
  */
-public class GradleInstallation extends ToolInstallation
-        implements EnvironmentSpecific<GradleInstallation>, NodeSpecific<GradleInstallation>, Serializable {
+public class GradleInstallation 
+        extends ToolInstallation
+        implements EnvironmentSpecific<GradleInstallation>, NodeSpecific<GradleInstallation> {
 
+    private static final long serialVersionUID = -1875114350592080381L;
```

Comment:
Why is this added if you removed `Serializable`? If it's still supposed to be serializable, you can keep the interface in the class declaration for explicitness.

Reply:
My IDE (eclipse) told me so ;) as the superclass ToolInstallation already implements Serializable i removed the duplicate specification. Is there any guideline concerning duplicate interface specifications?

Reply:
No guidelines that I can remember. It's more of a Java style thing in general.

Reply:
Okay you are right. i've added the Serializable interface!